doc/nrf/security/tfm.rst

```diff
@@ -27,7 +27,7 @@ The TF-M implementation in |NCS| is demonstrated in the following samples:
 
 * All :ref:`tfm_samples` in this SDK
 * All :ref:`cryptography samples <crypto_samples>` in this SDK
-* A series of :ref:`TF-M integration samples <zephyr:tfm_integration-samples>` available in Zephyr
+* A series of :zephyr:code-sample-category:`tfm_integration` available in Zephyr
```

Comment:
```suggestion
* A series of :zephyr:code-sample-category:`tfm_integration` examples available in Zephyr
```

Reply:
Added 'samples', think this is used more widely